src/librustdoc/html/render/mod.rs

```diff
@@ -765,13 +764,13 @@ fn assoc_const(
     _default: Option<&String>,
     link: AssocItemLink<'_>,
     extra: &str,
-    cx: &Context<'_>,
+    cx: &mut Context<'_>,
 ) {
     write!(
         w,
         "{}{}const <a href=\"{}\" class=\"constant\">{}</a>: {}",
         extra,
-        it.visibility.print_with_space(it.def_id, cx),
+        it.visibility.print_with_space(it.def_id, &cx.clone()),
```

Comment:
Why is this clone necessary? I think you can just do `&*cx`.

Reply:
Not sure, let me test

Reply:
Ah right, it differs the mutability and causes borrowing issues.

Reply:
What about moving this up and before the `write` so you can use `&mut cx` after?